deploy/olm-catalog/3scale-operator/0.0.1/3scale-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -2,7 +2,17 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   name: 3scale-operator.v0.0.1
-  namespace: openshift-operator-lifecycle-manager
+  namespace: placeholder
```

Comment:
The _placeholder_ namespace is replaced at runtime with the namespace where the operator is being installed.